util/xmlformat/xmlformat.conf

```diff
@@ -52,10 +52,10 @@ figure revnumber  # Fixme: move revnumber into inline after fixing wrong content
 # no entry break:
 para simpara subtitle
   entry-break = 0
-  element-break = 0
+  element-break = 1
 
 # Meta: usually 1 line blocks:
-title publishername colspec revremark term xi:include
+title titleabbrev publishername colspec revremark term xi:include
```

Comment:
Wo verwenden wir das eigentlich?

Reply:
Den `titleabbrev` habe ich in GUIMenus.xml eingesetzt um im HTML die Einträge im Tabellenverzeichnis besser zu gestalten. Ansonsten würden die `title`- Texte übernommen.

Also z.B. `<titleabbrev>&appname; - Datei-Menü</titleabbrev>` an Stelle von `<title>&appname; - Datei-Menü - Zugriff auf Datei- und Kontenoperation sowie Druckfunktion</title>` .

Reply:
Keine schlechte Idee!
Solltest du in
https://wiki.gnucash.org/wiki/Documentation_Improvement#Conventions
in der Nähe von `Table Titles` als `Chapter, … (structure) Titles`  o.ä. einfügen.

Reply:
@fellen : kannst du diesen PR mal durchwinken? Ich möchte die Formatierung mal wieder aktualisieren.